docs/openapi.yaml

```diff
@@ -927,6 +927,71 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /{publicationIdentifier}/file/{fileIdentifier}:
+    delete:
+      summary: Delete file
+      description: Deletes file metadata
```

Comment:
Does the endpoint delete the file itself (ultimately from S3) or the metadata we have stored about the file? This can be interpreted as either.

Reply:
Deletes metadata only (database entry). The plan is to have event handler which consumes "File deleted event" and removes it from S3.